File: 2015/js/day02/day02.js
const fs = require('fs');

console.log('Day 02 - Advent of Code 2015');

try {
    const data = fs.readFileSync('input.txt', 'utf8').split('\\n');
    console.log('Loaded ' + data.length + ' lines');
} catch (err) {
    console.log('No input.txt found.');
}

File: 2025/py/day02/day02.py
def main():
    print("Day 02 - Advent of Code 2025")
    try:
        with open('input.txt') as f:
            lines = f.read().splitlines()
            print(f"Loaded {len(lines)} lines")
    except FileNotFoundError:
        print("No input.txt found.")

if __name__ == '__main__':
    main()
